src/Tribe/Tickets.php

```diff
@@ -2392,9 +2392,11 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
 
 				$headers[] = sprintf(
 					'From: %1$s <%2$s>',
-					filter_var( $from_name, FILTER_SANITIZE_STRING ),
+					stripcslashes($from_name),
```

Comment:
We need to have empty spaces between the var when sending as a parameter
```suggestion
					stripcslashes( $from_name ),
```